.gitignore

```diff
@@ -167,7 +167,7 @@ DvcLiveLogger/
 *acc.tsv
 *loss.tsv
 *epoch.tsv
-*.png
+# *.png       //this may need to be added back but this is a temporary fix

```

Comment:
Add this line back,
but below it add
```suggestion
# *.png       //this may need to be added back but this is a temporary fix
!path/to/the/needed/pic.png
```